spec/nero/output/panoptes_api_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Nero::Output::PanoptesApi do
+  subject(:api) { described_class.new("http://example.org", "key", "secret") }
+
+  before do
+    stub_request(:post, "http://example.org/oauth/token")
+      .with(:body => {"client_id"=>"key", "client_secret"=>"secret", "grant_type"=>"client_credentials"},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.